rest-api/dao/base_dao.py

```diff
@@ -410,7 +417,7 @@ def _validate_update(self, session, obj, existing_obj):
     """
     if not existing_obj:
       raise NotFound('%s with id %s does not exist' % (self.model_type.__name__, id))
-    if existing_obj.version != obj.version:
+    if self.validate_version_match and existing_obj.version != obj.version:
```

Comment:
@robabram @wangy70 this is the `UpdatableDao`'s `version` behavior change we have discussed so far.